server/plugin/webhook.go

```diff
@@ -895,9 +895,22 @@ func (p *Plugin) handleCommentMentionNotification(event *github.IssueCommentEven
 		Message: message,
 		Type:    "custom_git_mention",
 	}
+	assignees := event.GetIssue().Assignees
 
 	for _, username := range mentionedUsernames {
 		// Don't notify user of their own comment
+		assigneeMentioned := false
+		for _, assignee := range assignees {
+			if username == *assignee.Login {
+				assigneeMentioned = true
+				break
+			}
+		}
+
+		if assigneeMentioned {
+			continue
+		}
```

Comment:
Suggestion to reorganize the newlines, and avoid using the temp var `assigneeMentioned` since it's unnecessary here

```suggestion

	assignees := event.GetIssue().Assignees
	for _, username := range mentionedUsernames {
		// Don't create duplicate notification for assignee
		for _, assignee := range assignees {
			if username == *assignee.Login {
				continue
			}
		}
```

Reply:
@sibasankarnayak Is this done for PRs or just issues here?

Also, I'm wondering if we instead want to do the inverse of this. In this case, the message will say "commented on a pull request you are assigned to" instead of "mentioned you in a comment". I'm not sure which provides more value.

@aaronrothschild I'd like your input here. Basically, there's currently a bug in the plugin if an assignee gets mentioned in a comment on GitHub, they currently get two notifications for the same comment. One for being mentioned and one for being an assignee.

It would complicate the code a bit to say "mentioned you on a pull request you are assigned to", so which notification message provides more value?

- "commented on a pull request you are assigned to"
- "mentioned you in a comment"